Dockerfile

```
@@ -14,14 +9,13 @@ RUN go mod download
 
 COPY . .
 ARG version
-ENV version=${version}
-RUN ./scripts/build-engine.sh
+RUN go build -mod=readonly -o ./bin/engine -ldflags="-X 'github.com/mesg-foundation/engine/version.Version=$version'" core/main.go
 
 FROM ubuntu:18.04
 RUN apt-get update && \
       apt-get install -y --no-install-recommends ca-certificates=20180409 && \
       apt-get clean && \
       rm -rf /var/lib/apt/lists/*
 WORKDIR /app
-COPY --from=build /project/engine .
-CMD ["./engine"]
+COPY --from=build /project/bin/engine .
+CMD ["./bin/engine"]
```

The binary is not in bin folder:
```suggestion
CMD [".engine"]
```